packages/web-components/update-scss-paths.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+declare -a component_names
+
```

Comment:
would it be useful to check for the dependency at the start? something like:

```suggestion

if ! command -v fswatch &>/dev/null; then
  echo "fswatch required (https://github.com/emcrisostomo/fswatch)"
  exit 1
fi

```

but with a more descriptive message

Reply:
I like that idea 👍